### ISSUE TITLE
Obsolete GO Terms present in the noctua-gpad products/annotations 'with' field

### ISSUE THREAD

**Issue by sierra-moxon** [geneontology/go-annotation#3933]
For example:
http://snapshot.geneontology.org/annotations/zfin.gpad.gz 
```
ZFIN    ZDB-GENE-980526-362    acts_upstream_of_or_within GO:0007155 ZFIN:ZDB-PUB-041022-3|PMID:15494018    ECO:0000305    GO:0005913    20041026   ZFIN      contributor=GOC:zfin_curators|model-state=production|noctua-model-id=gomodel:ZFIN_ZDB-GENE-980526-362
```

**Comment by ukemi** [comment]
We should add this as part of rule #20, shouldn't we? We should just check all GO identifiers in annotations.

**Comment by sierra-moxon** [comment]
@ukemi - check and correct(ie: use the replaced with value)?

**Comment by ukemi** [comment]
I would think so. Question is what do we do with "consider"? @pgaudet do these show up in the report for RUle 20?

**Comment by pgaudet** [comment]
Hi @ukemi 

'Consider' is not automatically repaired. Consider is when we want people to review. 


I found an example showing that this is behaving correctly in go-rules: 

> WARNING - Obsolete class with no replacement: Violates GORULE:0000020 (GO:0033955) -- GeneDB LmjF.36.1640:mRNA UMSBP GO:0033955 GO_REF:0000024 ISO GeneDB:Tb927.10.6060 P universal minicircle sequence binding protein, putative LmjF36.1640 transcript taxon:347515 20141124 GeneDB

here: http://release.geneontology.org/2021-09-01/reports/genedb_lmajor-report.html#gorule-0000020

[Term]
id: GO:0033955
def: "OBSOLETE. The process in which copies of the mitochondrial genome are distributed into daughter mitochondria upon mitochondrial fission." [GOC:mah]
property_value: term_tracker_item https://github.com/geneontology/go-ontology/issues/18417 xsd:anyURI
is_obsolete: true
consider: GO:0000002
consider: GO:0000266


Thanks, Pascale

**Comment by sierra-moxon** [comment]
@kltm - can you assign this to me?  I don't have permission to assign. thx in advance! :)

**Comment by ukemi** [comment]
@gaudet, yes I know that. But this ticket is looking at the 'with' field. I think the same behavior should occur and annotations with the consider tag would be reported just like they are if the obsolete term is the annotation object.

**Comment by pgaudet** [comment]
>  I think the same behavior should occur and annotations with the consider tag would be reported just like they are if the obsolete term is the annotation object

I agree

**Comment by pgaudet** [comment]
@sierra-moxon Was this fixed? I still see the merged term in 'with' field for that annotation in AmiGO: 



  | ctnnb1 | catenin (cadherin-associated protein), beta 1 |   | cell adhesion |   | ZFIN | Danio rerio | IC | GO:0005913 | armadillo segment polarity protein pthr45976 | protein_coding_gene |   | PMID:15494018 | 200410
-- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | --


[ctnnb1](http://amigo.geneontology.org/amigo/gene_product/ZFIN:ZDB-GENE-980526-362)	catenin (cadherin-associated protein), beta 1		[cell adhesion](http://amigo.geneontology.org/amigo/term/GO:0007155)		ZFIN	[Danio rerio](http://www.ncbi.nlm.nih.gov/Taxonomy/Browser/wwwtax.cgi?id=7955)	IC	[GO:0005913](http://amigo.geneontology.org/amigo/term/GO:0005913)	[armadillo segment polarity protein pthr45976](http://www.pantherdb.org/panther/lookupId.jsp?id=PTHR45976)	protein_coding_gene		[PMID:15494018](http://amigo.geneontology.org/amigo/reference/PMID:15494018)	200410
